roles/glusterfs/volume-delete/README.md

```diff
@@ -0,0 +1,76 @@
+volume-create
+=============
+
+Creates a GlusterFS volume. Volume can be of any type - replicate, distributed-replicate, arbiter ...
+
```

Comment:
Same as above. I think you haven't modified the documentation at all.

Reply:
Made changes to the documentation adding volume-delete role specifications.